### PR TITLE
Display map

### PR DESCRIPTION
**Pull request by NadineGrosskreuz**
Created a Leaflet map which shows the current location of the user if allowed
Many changes due to problems with Storybook, I am sorry for that!

### CODE REVIEW

**Review by aclaes — CHANGES_REQUESTED** [review]

File: src/components/Map/LocationMarker.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import { useState, useEffect } from 'react';
+import { Marker, Popup, useMapEvents } from 'react-leaflet';
+import 'leaflet/dist/leaflet.css';
+import 'leaflet-defaulticon-compatibility/dist/leaflet-defaulticon-compatibility.css';
+import 'leaflet-defaulticon-compatibility';
```
[the remark this answers is not in the document]
Comment (NadineGrosskreuz) [reply src/components/Map/LocationMarker.js:5]: I removed it!

File: src/components/Map/Map.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+const positions = [
+	{ name: 'hier ist was', lat: 52, long: 8 },
+	{ name: 'hier ist was anderes', lat: 51, long: 9 },
+];
+
+export default function Map() {
+	return (
+		<MapContainer
+			scrollWheelZoom
+			style={{ height: '530px', width: '350px' }}
```
[the remark this answers is not in the document]
Comment (NadineGrosskreuz) [reply src/components/Map/Map.js:16]: I have set a z-index on my footer, but it is not working. Can you explain why?

[the remark this answers is not in the document]
Comment (aclaes) [reply src/components/Map/Map.js:16]: The `z -index` value for the navigation bar must be higher than the map. It seems the map uses:

```
.leaflet-tile-pane {
  z-index: 200;
}
```

[the remark this answers is not in the document]
Comment (NadineGrosskreuz) [reply src/components/Map/Map.js:16]: Ah I see, there are even more z-indexes. So with a very high z-index it works, thanks! :)

**Review by aclaes — APPROVED** [review]

**Pull request merged**